.gitignore

```diff
@@ -212,3 +215,6 @@ PSScriptAnalyzer/
 
 # Test result file
 TestResults.xml
+
+# CrossCompatibility submodule
```

Comment:
I thought at first this was referring to a git submodule, maybe just `module` is easier and less ambiguous
```suggestion
# CrossCompatibility module
```